FrameworkSystem/private/standardLogging/Backend/Backend.py

```diff
@@ -0,0 +1,53 @@
+"""
+Backend wrapper
+"""
+
+__RCSID__ = "$Id$"
+
+from abc import ABCMeta, abstractmethod
```

Comment:
More as an abstract class, with the possibility to add code in some methods to factorize without anyone can use them. 
If I transform it as an interface, I have to delete the code in the getHandler and setFormat methods, and I have to duplicate this code in each subclass. Are you sure ?